app/main.py

```diff
@@ -1,3 +1,2 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    return phrase.lower().count(letter.lower())
```

Comment:
The function `count_occurrences` is expected to return an integer, but the current implementation returns a float. The `count` method returns an integer, so there is no need for division. Simply return the result of `phrase.lower().count(letter.lower())` to ensure the function returns an integer as intended.